packages/quantic/force-app/main/default/lwc/quanticSort/__tests__/quanticSort.test.js

```diff
@@ -5,6 +5,13 @@ import QuanticSort from 'c/quanticSort';
 import {createElement} from 'lwc';
 import * as mockHeadlessLoader from 'c/quanticHeadlessLoader';
 
+const selectors = {
+  lightningCombobox: 'lightning-combobox',
+  componentError: 'c-quantic-component-error',
+  sortDropdown: '[data-cy="sort-dropdown"]',
```

Comment:
In order to not forget it, can you please make this change, and also in the `.html` file of `quanticSort`?

```suggestion
  sortDropdown: '[data-testid="sort-dropdown"]',
```

the `data-testid` selectors are the new selectors for playwright where the `data-cy` were meant for cypress.
So we try to update at least the ones we need in the tests, so when we remove cypress, we can remove all the `data-cy` everywhere knowing all the ones we use will have been renamed to `data-testid` :) 

It also lets you use the `page.getByTestId` method from Playwright.